service-manual/assets/toolkit

```diff
@@ -1 +1 @@
-Subproject commit 6f9febc4204f0ffbf83738c0c65af55f5f795faa
+Subproject commit 5199b77ee98f74c9815725ba8aafdbad699bc6df
```

Comment:
You seem to have rolled back to an older version of the frontend toolkit. Was that intentional?